tests/functional/strategy/test_misc.py

```diff
@@ -131,10 +131,6 @@ def test_sweep(gov, vault, strategy, rando, token, other_token):
     with brownie.reverts("!shares"):
         strategy.sweep(vault.address, {"from": gov})
 
-    # Protected token doesn't work
-    with brownie.reverts("!protected"):
-        strategy.sweep(strategy.protectedToken(), {"from": gov})
-
```

Comment:
Please keep this functionality

Reply:
https://github.com/yearn/yearn-vaults/pull/244#discussion_r593956110